From 37d06227a113f728001452c105112470b80e7c77 Mon Sep 17 00:00:00 2001
From: Guoqing Jiang <guoqing.jiang@suse.com>
Date: Mon, 28 Mar 2022 14:19:33 +0800
Subject: [PATCH] Revert "dmaengine: idxd: fix delta_rec and crc size field for
 completion record"
Patch-mainline: Never, revert stable patch which is added again in sorted section
References: bsn#161
Modified-by-SEL: No


This reverts commit 0e3f14755111522018c0e325d1ce768dd609641a.

Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 include/uapi/linux/idxd.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/uapi/linux/idxd.h b/include/uapi/linux/idxd.h
index 9d9ecc0f4c38..fdcdfe414223 100644
--- a/include/uapi/linux/idxd.h
+++ b/include/uapi/linux/idxd.h
@@ -187,8 +187,8 @@ struct dsa_completion_record {
 			uint32_t	rsvd2:8;
 		};
 
-		uint32_t	delta_rec_size;
-		uint32_t	crc_val;
+		uint16_t	delta_rec_size;
+		uint16_t	crc_val;
 
 		/* DIF check & strip */
 		struct {
-- 
2.26.2

